app/helpers/application_helper.rb

```diff
@@ -102,7 +102,7 @@ def render_map(lat, lon)
   def title_suggestion(comment)
     comment.body.gsub(/\[propose:title\](.*?)\[\/propose\]/) do
       a = ActionController::Base.new
-      is_creator = current_user.drupal_user == Node.find(comment.nid).author
+      is_creator = @current_user.drupal_user == Node.find(comment.nid).author
```

Comment:
I think we need to remove the `@` here no? Some of the tests are failing because `current_user` is undefined.